app/presenters/concerns/deferred_offer_api_data.rb

```diff
@@ -0,0 +1,12 @@
+module DeferredOfferAPIData
+  def schema
+    super.deep_merge!({
+      attributes: {
+        offer: {
```

Comment:
while testing this locally, I noticed we are setting the offer object regardless of the status. This means that instead of getting a `null` offer object we are setting the following attributes on each application. Could we look to only set those values on applications past the offer stage, and allow for `offer_deferred_at` to be `nil`: `application_choice.offer_deferred_at&.iso8601`

Reply:
updated